keyboards/pabile/p18/config.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0000
+#define PRODUCT_ID      0x0000
```

Comment:
These should be set to something.

Reply:
Thank you. I have updated the original file.